proseco/fid.py

```diff
@@ -474,6 +507,9 @@ def set_spoilers_score(self, fid):
             elif np.any(yellow):
                 fid["spoiler_score"] = 1
 
+            if fid_trap_spoiler:
+                fid["spoiler_score"] += 10
```

Comment:
Here I've injected fid trap spoilers as a new thing with a new number.  This is all arbitrary.